CHANGELOG.rst

```diff
@@ -11,6 +11,13 @@ Change Log
 
 .. There should always be an "Unreleased" section for changes pending release.
 
+Unreleased
+----------
+Changed
+~~~~~~~
+* Remove unused ``background_task`` monitoring function.
+* Remove ``get_current_transaction`` from the public API.
```

Comment:
[question]: Do we want to make this a 7.0.0 breaking change because that is how the contract phase is typically handled (i.e. we get things so that they are unused, and then we remove them)? It seems right, even though it is strange. Either way, I think we need to decide and do it as part of this PR, rather than leaving this to some future person to decide. Thoughts?

```suggestion
Removed
~~~~~~~
* Remove unused ``background_task`` monitoring function.
* Remove ``get_current_transaction`` (used internally only) from the public API.
```

Reply:
Yeah, I thought these changes would be more minor than they ended up being. I'll bump the version number so we release it as 7.0.0.